Sources/Casbin/APi/ManagementApi.swift

```diff
@@ -106,13 +106,13 @@ public extension CoreApi {
         model.hasPolicy(sec: "g", ptype: ptype, rule: params)
     }
     func getAllNamedSubjects(ptype:String) -> [String] {
-        model.getValuesForFieldInPolicy(sec: "p", ptype: ptype, fieldIndex: 0)
+        model.getValuesForFieldInPolicy(sec: "p", ptype: ptype, fieldIndex: model.getModel()["p"]?["p"]?.tokens.firstIndex(of: "p_sub") ?? 0)
     }
     func getAllNamedObjects(ptype:String) -> [String] {
-        model.getValuesForFieldInPolicy(sec: "p", ptype: ptype, fieldIndex: 1)
+        model.getValuesForFieldInPolicy(sec: "p", ptype: ptype, fieldIndex: model.getModel()["p"]?["p"]?.tokens.firstIndex(of: "p_obj") ?? 1)
     }
     func getAllNamedActions(ptype:String) -> [String] {
-        model.getValuesForFieldInPolicy(sec: "p", ptype: ptype, fieldIndex: 2)
+        model.getValuesForFieldInPolicy(sec: "p", ptype: ptype, fieldIndex: model.getModel()["p"]?["p"]?.tokens.firstIndex(of: "p_act") ?? 2)
```

Comment:
it's nice!
Please modify the DefaultModel.getValuesForFieldInPolicy(sec:String,ptype:String,fieldIndex:Int)  function at the same time.
```swift
      public func getValuesForFieldInPolicy(sec:String,ptype:String,fieldIndex:Int) -> [String]  {
            let policy = getPolicy(sec: sec, ptype: ptype).reduce(Set<String>()) { acc, x in
            var acc = acc
            acc.insert(x[fieldIndex])
            return acc
        }
        return Array(policy)
    }
```